daprdocs/content/en/developing-applications/sdks/_index.md

```diff
@@ -24,12 +24,12 @@ Select your [preferred language below]({{< ref "#sdk-languages" >}}) to learn mo
 |----------|:------|:----------:|:-----------:|:---------:|:---------:|
 | [.NET]({{< ref dotnet >}}) | Stable | ✔ |  [ASP.NET Core](https://github.com/dapr/dotnet-sdk/tree/master/examples/AspNetCore) | ✔ | ✔ |
 | [Python]({{< ref python >}}) | Stable | ✔ | [gRPC]({{< ref python-grpc.md >}}) <br />[FastAPI]({{< ref python-fastapi.md >}})<br />[Flask]({{< ref python-flask.md >}})| ✔ | ✔ |
-| [Java]({{< ref java >}}) | Stable | ✔ | Spring Boot | ✔ | |
-| [Go]({{< ref go >}}) | Stable | ✔ | ✔ | ✔ | |
+| [Java]({{< ref java >}}) | Stable | ✔ | Spring Boot | ✔ | ✔ |
```

Comment:
```suggestion
| [Java]({{< ref java >}}) | Stable | ✔ | Spring Boot  <br /> Quarkus| ✔ | ✔ |
```
I suggest also have Quarkus given samples like this 
https://github.com/mcruzdev/pizza-quarkus/commit/13ad7f4009379a228e8281c9039b6172604d0909

Reply:
Also worth adding a link to Java samples for Spring Boot?